src/feature_melspectrogram.py

```diff
@@ -0,0 +1,117 @@
+from essentia.streaming import MonoLoader, TensorflowInputMusiCNN, TensorflowInputVGGish, FrameCutter
+import essentia.standard as es
+from essentia import Pool, run, reset
+import numpy as np
+
+
+class MelSpectrogramMusiCNN():
+    def __init__(self):
+        self.sample_rate = 16000
+        self.hop_size = 256
+        self.frame_size = 512
+
+        self.pool = Pool()
+        self.loader = MonoLoader(sampleRate=self.sample_rate)
+        self.frameCutter = FrameCutter(frameSize=self.frame_size, hopSize=self.hop_size)
+        self.mels = TensorflowInputMusiCNN()
+
+        self.loader.audio >> self.frameCutter.signal
+        self.frameCutter.frame >> self.mels.frame
+        self.mels.bands >> (self.pool, 'mel_bands')
```

Comment:
`>>` 🤔

Reply:
This is the `connect` operator for [Essentia streaming](https://essentia.upf.edu/streaming_architecture.html#essentia-streaming-mode-architecture) algorithms

Reply:
But is this valid python code? It looks like it was simply copied from C++!

Reply:
Yes, this is how Essentia is used in streaming mode in Python. 
Maybe [this](https://essentia.upf.edu/essentia_python_tutorial.html#using-essentia-in-streaming-mode) is the best resource to get started with it in case you want to learn more 🙂